src/api/v2/endpoint/chat.py

```
@@ -0,0 +1,97 @@
+            isSuccess=True,
+            code="AI200",
+            message="[v2] 비교 분석 성공입니다.",
+            result=compare_result,
+        )
+
+    except Exception as e:
+        logger.exception("[v2] Compare endpoint error")
+        return CommonResponse(
+            isSuccess=False,
+            code="AI500",
+            message=f"비교 분석 중 오류 발생: {str(e)}",
```

Ruff (0.15.2)

[warning] Use explicit conversion flag

Replace with conversion flag

(RUF010)